nupic/algorithms/CLAClassifier.py

```diff
@@ -213,6 +213,31 @@ def __setstate__(self, state):
     self._version = BitHistory.__VERSION__
 
 
+  def write(self, proto):
+    proto.id = self._id
+
+    statsProto = proto.init("stats", len(self._stats))
```

Comment:
Actually, never mind.  At least not without modifying the BitHistory.capnp schema to match.  @scottpurdy What do you think?